docker/add-user.sh

```diff
@@ -30,11 +30,7 @@ ENCPASS=$(doveadm pw -u "${EMAIL}" -p "${PASSWORD}")
 # Edit dovecot users: remove user if it exits.
 mkdir -p /data/dovecot
 touch /data/dovecot/users
-if grep -q "^${EMAIL}:" /data/dovecot/users; then
-	cp /data/dovecot/users /data/dovecot/users.old
-	grep -v "^${EMAIL}:" /data/dovecot/users.old \
-		> /data/dovecot/users
-fi
+sed -i "/^${EMAIL}:/d" /data/dovecot/users
```

Comment:
Can you use `sed --in-place=.old` instead of `sed -i`, so it leaves the previous file around just in case there are issues?

Thank you!

Reply:
Done